docs/streams/upgrade-guide.html

```diff
@@ -141,6 +141,22 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
 
     <h3><a id="streams_api_changes_400" href="#streams_api_changes_400">Streams API changes in 4.0.0</a></h3>
 
+    <p>
+        In this release, eos-v1 (Exactly Once Semantics version 1) is no longer supported. Additionally, all deprecated methods, classes, APIs, and config parameters
+        up to (including) AK 3.5 release have been removed.
```

Comment:
```suggestion
        up to (including) Kafka Streams 3.6 release have been removed.
```

Reply:
We keep 3 release of history, so 3.9/3.8/3.7, right?

I guess, we did not deprecate anything in 3.6, so nothing needed to be removed, but I would still say 3.6 here?